sqlflow_models/lstmclassifier.py

```diff
@@ -10,6 +12,7 @@ def __init__(self, feature_columns, stack_units=[32], hidden_size=64, n_classes=
         :param n_classes: Target number of classes.
         :type n_classes: int.
         """
+        global _loss
```

Comment:
Why are these `global` keywords needed?

Reply:
Please, check out the above comment:

`_train_lr` may be configured by SQL statements when initializing the model instance. e.g.

```
SELECT * FROM mytrain_table TO TRAIN sqlflow_models.DNNClassifier WITH train_lr=0.0001 ...
```

In the above SQL statement, `train_lr` will be passed to `DeepEmbeddingClusterModel` to configure the learning rate.